Automattic/vip-go-mu-plugins
# Add wp-parsely 3.5

## Description

This PR adds `wp-parsely` version 3.5.0 to `mu-plugins`. After merging this PR, sites can update to the 3.5.0 version using the following code:

```
add_filter(
	'wpvip_parsely_version',
	function() {
		return '3.5';
	} 
);

add_filter( 'wpvip_parsely_load_mu', '__return_true' );
```
Another PR that pins this version as the default will be created in the future.


## Changelog Description

### Add wp-parsely 3.5.0

We added wp-parsely 3.5.0 plugin as a (non-default) option to sites.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Add the snippet provided in the description to enable the plugin and the 3.5 version.
